omniauth-shopify-oauth2.gemspec

```diff
@@ -16,8 +16,8 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
   s.require_paths = ['lib']
 
-  s.add_runtime_dependency 'omniauth-oauth2', '~> 1.2'
-  s.add_runtime_dependency 'oauth2', '~> 1.1.0'
+  s.add_runtime_dependency 'omniauth-oauth2', '~> 1.4.0'
```

Comment:
updated to lock this version to depend on 2.1.9 and greater, mirroring omniauth gem's requirements